### PR TITLE
Add builder to LocationPickerActivity

### PR DESCRIPTION
**Pull request by marcserrascmspain**

### CODE REVIEW

File: leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java [hunk 1]
```diff
@@ -1007,4 +1007,87 @@ private void closeKeyboard() {
...
+    private String locationSearchZone;
+    private String layoutsToHide = "";
+    private boolean enableSatelliteView = true;
+    private boolean shouldReturnOkOnBackPressed = false;
+    private List<LekuPoi> lekuPois;
+
+    public Builder() {
+    }
+
+    public Builder withLatitude(double latitude) {
```
Comment (alorma) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: location for builder should be a simple `withLocation()`... why?

Lib cannot work without both params, so it's weird to set `location` or `longitude` ...

Comment (marcserrascmspain) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: It's only to expose bundle params (that neither ensure to fill the two parameters), but ok, we can unify the Builder setter `withLocation(double latitude, double longitude)`

Comment (alorma) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: 👍

File: leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java [hunk 2]
```diff
@@ -1007,4 +1007,83 @@ private void closeKeyboard() {
...
+      this.layoutsToHide = String.format("%s|%s", layoutsToHide, OPTIONS_HIDE_CITY);
+      return this;
+    }
+
+    public Builder withZipCodeHidden() {
+      this.layoutsToHide = String.format("%s|%s", layoutsToHide, OPTIONS_HIDE_ZIPCODE);
+      return this;
+    }
+
+    public Builder withLekuPois(List<LekuPoi> pois) {
```
Comment (gerardpedreny) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: Maybe I would say only "withPois", Leku is not necessary

**Pull request merged**